src/App/Components/ArticleNotFound/ArticleNotFound.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Navbar from '../Common/NavProfile/navbar';
+
+const ArticleNotFound = () => (
```

Comment:
@MCFrank16 why not have a generic `NotFound` component and anyone can pass in the entity not found (i.e user/article/comment etc) in as a prop?

Reply:
that would be a good idea, lemme refactor this